src/components/DrawerSection.tsx

```diff
@@ -116,9 +129,13 @@ export const DrawerSectionCollapsibleEntry = ({
                 cursor="pointer"
                 onClick={onToggle}
             >
-                <Text whiteSpace="nowrap" width="120px" flexShrink="0">
-                    {capitalize(title)}
-                </Text>
+                {icon}
```

Comment:
Not your code directly, but you may want to set `alignItems` to center now we have the icons (see screenshot with `flex-start`)

<img width="782" alt="Capture d’écran 2022-09-22 à 16 58 31" src="https://user-images.githubusercontent.com/44287876/191782336-5f87d518-b56d-4170-ac64-ef08b8cb6667.png">

Reply:
nice catch